cellprofiler/modules/loadimages.py

```diff
@@ -436,199 +463,241 @@ def example_path_fn():
+syntax can be used to name different parts of your expression. The
+syntax *(?P<fieldname>expr)* will extract whatever matches *expr* and
+assign it to the measurement,\ *fieldname* for the image.
+
+For instance, a researcher uses plate names composed of a string of
+letters and numbers, followed by an underscore, then the well,
+followed by another underscore, followed by an “s” and a digit
+representing the site taken within the well (e.g.,
+*TE12345_A05_s1.tif*). The following regular expression will capture
+the plate, well, and site in the fields “Plate”, “Well”, and “Site”:
+
++----------------------------------------------------------------+------------------------------------------------------------------+
+| ^(?P<Plate>.*)_(?P<Well>[A-P][0-9]{1,2})_s(?P<Site>[0-9])   |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| ^                                                              | Start only at beginning of the file name                         |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| (?P<Plate>                                                     | Name the captured field *Plate*                                  |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| .*                                                            | Capture as many characters as follow                             |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| _                                                             | Discard the underbar separating plate from well                  |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| (?P<Well>                                                      | Name the captured field *Well*                                   |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| [A-P]                                                          | Capture exactly one letter between A and P                       |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| [0-9]{1,2}                                                     | Capture one or two digits that follow                            |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| _s                                                            | Discard the underbar followed by *s* separating well from site   |
```

Comment:
Please add 1 more space in the first column of this line  to get the table to format correctly.